buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -55,4 +55,96 @@ interface FladleConfig {
+   * The shards are run in parallel on separate devices. For example,
+   * if your test execution contains 20 test cases and you specify four shards, each shard executes five test cases.
+   * The number of shards should be less than the total number of test cases.
+   * The number of shards specified must be >= 1 and <= 50.
+   * This option cannot be used along max-test-shards and is not compatible with smart sharding.
+   * If you want to take benefits of smart sharding use max-test-shards instead.
+   * (default: null)
+   */
+  var numUniformShards: Int?
+
+  /**
+   * A key-value map of additional details to attach to the test matrix.
+   * Arbitrary key-value pairs may be attached to a test matrix to provide additional context about the tests being run.
+   * When consuming the test results, such as in Cloud Functions or a CI system,
+   * these details can add additional context such as a link to the corresponding pull request.
+   */
+  var clientDetails: Map<String, String>
+
+  /**
+   * Always run - these tests are inserted at the beginning of every shard
+   * useful if you need to grant permissions or login before other tests run
+   */
+  var testTargetsAlwaysRun: List<String>
+
+  /**
+   * A list of device-path: file-path pairs that indicate the device paths to push files to the device before starting tests, and the paths of files to push.
+   * Device paths must be under absolute, whitelisted paths (${EXTERNAL_STORAGE}, or ${ANDROID_DATA}/local/tmp).
+   * Source file paths may be in the local filesystem or in Google Cloud Storage (gs://…).
+   */
+  var otherFiles: Map<String, String>
+
+  /**
+   * The name of the network traffic profile, for example LTE, HSPA, etc,
+   * which consists of a set of parameters to emulate network conditions when running the test
+   * (default: no network shaping; see available profiles listed by the `flank test network-profiles list` command).
+   * This feature only works on physical devices.
+   */
+  var networkProfile: String?
+
+  /**
+   * The path to a Robo Script JSON file.
+   * The path may be in the local filesystem or in Google Cloud Storage using gs:// notation.
+   * You can guide the Robo test to perform specific actions by recording a Robo Script in Android Studio and then specifying this argument.
+   * Learn more at [https://firebase.google.com/docs/test-lab/robo-ux-test#scripting].
+   */
+  var roboScript: String?
+
+  /**
+   * List of robo_directives that you can use to customize the behavior of Robo test.
+   * The type specifies the action type of the directive, which may take on values click, text or ignore.
+   * Each directive is list of String = [type, key, value]
+   * Each key should be the Android resource name of a target UI element and each value should be the text input for that element.
+   * Values are only permitted for text type elements, so no value should be specified for click and ignore type elements.
+   */
+  var roboDirectives: List<List<String>>
```

Comment:
ohhh. do we want this to be a `List<Triple>`?
Or is it better represented as `Map<String, List<String>>`?

Reply:
I was thinking about that but I wanted to allow user to skip 3rd parameter.
```
roboDirectives = [
      ["click", "button1", ""],
      ["ignore", "button2"],  <-- 3rd skipped
      ["text", "field1", "my text"],
    ]
```
`Triple` will force user to always provide three strings. But I am open to suggestions